pkg/plugin/admissioncontrollers/webhook/admission_controller.go

```diff
@@ -184,6 +239,13 @@ func (c *admissionController) mutate(req *admissionv1.AdmissionRequest) *admissi
 		return admissionResponseBuilder(uid, false, err.Error(), nil)
 	}
 
+	if annotation, ok := pod.Annotations[userInfoAnnotation]; ok && !c.bypassAuth {
+		if err := c.validateExternalUserInfo(req, annotation); err != nil {
+			log.Logger().Error("external authentication failed", zap.Error(err))
```

Comment:
The message is not correct. The external authentication did not fail. The `real` user is not allowed to set the annotations. Message should be more like: "User XX is not allowed to set user annotation".

Reply:
Corrected in both places